source/developers/cook-books/how-tos/migrate-site-to-elasticsearch.rst

```diff
@@ -43,41 +43,113 @@ Updating to Elasticsearch
 
 In case you decide to update your site to use Elasticsearch instead of Solr you can follow these steps:
 
-1. Find all references to ``searchService`` in your FreeMarker templates and move them to Groovy scripts
-2. Find all references to ``searchService`` in your Groovy scripts and replace them with the Elasticsearch service
+#. Overwrite the target in the Deployer to use Elasticsearch instead of Solr
+#. Index all existing content in Elasticsearch
+#. Find all references to ``searchService`` in your FreeMarker templates and replace them with the Elasticsearch service
+#. Find all references to ``searchService`` in your Groovy scripts and replace them with the Elasticsearch service
+#. Delete the unused Solr core if needed (can be done using the Solr Admin UI or the ``data/indexes`` folder)
+
+^^^^^^^^^^^^^^^^^^^^
+Overwrite the target
+^^^^^^^^^^^^^^^^^^^^
+
+For authoring environments:
+
+.. code-block:: bash
+  :linenos:
+
+  curl --request POST \
+    --url http://DEPLOYER_HOST:DEPLOYER_PORT/api/1/target/create \
+    --header 'content-type: application/json' \
+    --data '{
+      "env": "preview",
+      "site_name": "SITE_NAME",
+      "template_name": "local",
+      "repo_url": "INSTALL_DIR/data/repos/sites/SITE_NAME/sandbox",
+      "disable_deploy_cron": true,
+      "replace": true
+    }'
+
+For delivery environments:
+
+.. code-block:: bash
+  :linenos:
+
+  curl --request POST \
+    --url http://DEPLOYER_HOST:DEPLOYER_PORT/api/1/target/create \
+    --header 'content-type: application/json' \
+    --data '{
+      "env": "default",
+      "site_name": "SITE_NAME",
+      "template_name": "remote",
+      "repo_url": "INSTALL_DIR/data/repos/sites/SITE_NAME/published",
+      "repo_branch": "live",
+      
+      ... any additional settings like git credentials ...
```

Comment:
Should we point to the doc that specifies what other additional settings you can send?